### ISSUE TITLE
[Quest][Classic][Westfall]Legend of Stalvan

### ISSUE THREAD

**Issue by Akkalevil** [dalaranwow/dalaran-wow#1466]
Minor bug : the Forlorn spirit is neutral instead of hostile when it appears.

**Comment by the-isolation-diode** [comment]
bugged  :  in the legend of stalvan when you are in eastvale logging camp about to go upstairs to check the chest, you kill the spirit in the room and theres no chest, you go back down and drop/reissue quest and go up and chest is there

**Comment by Rushor** [comment]
http://wotlk.openwow.com/quest=75

**Comment by Rushor** [comment]
DB/Quest: The Legend of Stalvan - Several updates

* Forlorn Spirit is hostile now
* Forlorn Spirit attacks the player o spawn
* Forlorn Spirit correctly displays the name of the player in the aggrotext
* removed the respawntime of Marshal Haggard's Chest (it respawns instantly now)
* closes https://github.com/dalaranwow/dalaran-wow/issues/1466
* commit: dfb2836773df0ea41c0bf8a3d382fa820916db90

![GitHub Logo](http://i.imgur.com/knJSjTG.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
Game/SmartScripts: Implement SMART_ACTION_CHECK_DUPLICATE

* this action allows to check for specific entries of creatures in range. If there is a duplicate of it, it will despawn 
* usefull for quests like 'The Legend of Stalvan', there only one Forlorn Spirit should spawn
* updates https://github.com/dalaranwow/dalaran-wow/issues/1466
* commit: 3575dad02c2e31ef41b3841a7aeb488421001f09
